framework/src/steppers/DT2Stepper.C

```diff
@@ -0,0 +1,146 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "DT2Stepper.h"
+#include "FEProblem.h"
+#include "Transient.h"
+#include "MooseApp.h"
+
+template<>
+InputParameters validParams<DT2Stepper>()
+{
+  InputParameters params = validParams<Stepper>();
+
+  params.addRequiredParam<Real>("dt", "The initial time step size.");
+  params.addRequiredParam<Real>("e_tol", "Target error tolerance.");
+  params.addRequiredParam<Real>("e_max", "Maximum acceptable error.");
+  params.addParam<Real>("max_increase", 1.0e9, "Maximum ratio that the time step can increase.");
```

Comment:
Don't know.  It was copied straight out of `DT2.C`: https://github.com/idaholab/moose/blob/devel/framework/src/timesteppers/DT2.C#L39

I'll switch it to `max()`